tests/utils/dir-is-empty.test.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+const dirToCreate = path.join(process.cwd(), 'temp');
```

Comment:
Use `os.tmpdir()` here instead of `process.cwd()` and use the name of the of the test as the directory, e.g. `dir_is_empty`.